_data/projects/IcebergAccountingSuite.yml

```diff
@@ -0,0 +1,19 @@
+name: Iceberg Accounting Suite
+desc: Iceberg Accounting Suite is intended to be a full service accounting suite capable of running small business applications such as invoicing, inventory, a cash drawer, card readers and POS compatability, website connectivity, and more. Written in Python using PySimpleGUI, Iceberg currently allows users to create an encrypted database and journal transactions against a set of accounts with vendor tracking.
+site: https://github.com/josephmbasile/IcebergAccountingSuite
+tags:
+  - accounting
+  - bookkeeping
+  - pos
+  - kms
+  - card
+  - reader
+  - register
+  - inventory
+  - vendor
+  - customer
+  - accounts
+  - books
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/users/josephmbasile/projects/1
```

Comment:
Project boards are not supported currently - we'd need a way to compute the number of cards in the project, which requires interacting with the GraphQL API...

Reply:
Hello thanks for helping me to get this working. Do you have a suggestion for something else I should use instead of the project board? Specifically for that link field. It was the only thing I could figure out how to add tags to the issues. I'm still new to GitHub so bear with me, thanks.

Reply:
![image](https://github.com/user-attachments/assets/6d021d8f-1010-4091-9824-794c7095b3b9)

Does this work? I'll leave it like this until you tell me otherwise. Happy new year!